### DISCUSSION TITLE
Pocketbase with Sequelize ORM?

### DISCUSSION THREAD

**Discussion by mihkuno** [pocketbase/pocketbase#3810] [Q&A] [answered]
I use Object-Relational Mapping to create my databases it does support sqlite. Problem is, how do i connect the database with it?

**Comment by ganigeorgiev** [comment, marked as answer]
The database file is located in `pb_data/data.db` and while technically you can open it from external processes I don't recommend it unless you really know what you are doing.

If you just want to use an ORM with SQLite, then you may not need PocketBase and prefer using SQLite directly as part of your node/deno/bun/etc. process.

There is really very little to no benefit of using PocketBase if you are not going to make use of its web APIs or db abstraction.
